android/library/src/main/java/com/polidea/multiplatformbleadapter/BleAdapter.java

```diff
@@ -88,7 +93,7 @@ void getCharacteristicsForDevice(
             OnErrorCallback onErrorCallback);
 
     void getCharacteristicsForService(
-            String serviceIdentifier,
+            int serviceIdentifier,
```

Comment:
Well, it's not UUID. UUID is not something which uniquely identifies service, characteristic or descriptor. UUID is more of a class or a type. Internally, what differentiate above objects are their handles in integer form, to which we don't have access on iOS/Android. I stick to integer form as it is more efficient/straightforward to pass and compare on different frontends. Also, it's harder to mix UUID with identifiers, because both have different types inside the implementation.